### ISSUE TITLE
Configurable HTTP method

### ISSUE THREAD

**Issue by brianstorti** [rmosolgo/graphiql-rails#2]
In case we want to use an http verb other than `POST`, it'd be useful to be able to specify it when we mount the engine. Something like:

```ruby
mount GraphiQL::Rails::Engine, at: "/graphiql", graphql_path: "/graph", method: :get
```

If you thing this is useful, I can work on the change.

Thanks.

**Comment by rmosolgo** [comment]
Yes, that'd be great! Can we call it `graphql_method`? That way it matches `graphql_path` & it's clear that it's the method for the GraphQL API endpoint, not for the GraphiQL editor.
